console/backend/src/main/java/org/frankframework/util/RequestUtils.java

```diff
@@ -23,16 +23,14 @@
 
 import jakarta.annotation.Nonnull;
 import jakarta.annotation.Nullable;
-
+import lombok.NoArgsConstructor;
 import org.apache.commons.lang3.StringUtils;
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
 import org.frankframework.management.web.ApiException;
 import org.springframework.http.MediaType;
 import org.springframework.web.multipart.MultipartFile;
 
-public abstract class RequestUtils {
-	private static final Logger LOG = LogManager.getLogger(RequestUtils.class);
+@NoArgsConstructor(access = lombok.AccessLevel.PRIVATE)
```

Comment:
Yes exactly. It is an util class right, so that's the purpose, to not make an instance. The previous `abstract` definition did the same, in a dirty way.